protocol/headers_opts_test.go

```diff
@@ -0,0 +1,221 @@
+// Copyright (c) 2020 Contributors to the Eclipse Foundation
```

Comment:
The copyright year of new added files mist be the creation year.
Please adjust in all added files accordingly.